components/american-opto/L196L.stanza

```diff
@@ -0,0 +1,39 @@
+#use-added-syntax(jitx)
+defpackage ocdb/american-opto/L196L:
+  import core
+  import collections
+  import math
+  import jitx
+  import jitx/commands
+  import ocdb/defaults
+
+  import ocdb/land-patterns
+  import ocdb/symbols
+  import ocdb/box-symbol
+  import ocdb/bundles
+  import ocdb/generator-utils
+  import ocdb/generic-components
+
+;<test>
```

Comment:
@m-hilgendorf , in another PR, @PhilippeFerreiraDeSousa wants to remove these comments

Reply:
These are still necessary, and the tests will break in this branch without the annotations.

Reply:
The other PR changes the fallback behaviour if there is no annotation of argument values:
- tries to guess the argument value of the macro on its own
- else skip it